sample-extension-plugin/src/test/java/org/opensearch/jobscheduler/sampleextension/SampleJobRunnerIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.jobscheduler.sampleextension;
+
+import org.junit.Assert;
+import org.opensearch.jobscheduler.spi.schedule.IntervalSchedule;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+public class SampleJobRunnerIT extends SampleExtensionIntegTestCase {
+
+    public void testJobCreateWithCorrectParams() throws IOException {
+        SampleJobParameter jobParameter = new SampleJobParameter();
+        jobParameter.setJobName("sample-job-it");
+        jobParameter.setIndexToWatch("http-logs");
+        jobParameter.setSchedule(new IntervalSchedule(Instant.now(), 1, ChronoUnit.MINUTES));
+        jobParameter.setLockDurationSeconds(120L);
+
+        // Creates a new watcher job.
+        String jobId = OpenSearchRestTestCase.randomAlphaOfLength(10);
+        SampleJobParameter schedJobParameter = createWatcherJob(jobId, jobParameter);
+
+        // Asserts that job is created with correct parameters.
+        Assert.assertEquals(jobParameter.getName(), schedJobParameter.getName());
+        Assert.assertEquals(jobParameter.getIndexToWatch(), schedJobParameter.getIndexToWatch());
+        Assert.assertEquals(jobParameter.getLockDurationSeconds(), schedJobParameter.getLockDurationSeconds());
+    }
+
+    public void testJobDeleteWithDescheduleJob() throws Exception {
+        String index = createTestIndex();
+        SampleJobParameter jobParameter = new SampleJobParameter();
+        jobParameter.setJobName("sample-job-it");
+        jobParameter.setIndexToWatch(index);
+        jobParameter.setSchedule(new IntervalSchedule(Instant.now(), 1, ChronoUnit.MINUTES));
+        jobParameter.setLockDurationSeconds(120L);
+
+        // Creates a new watcher job.
+        String jobId = OpenSearchRestTestCase.randomAlphaOfLength(10);
+        SampleJobParameter schedJobParameter = createWatcherJob(jobId, jobParameter);
+
+        // wait till the job runner runs for the first time after 1 min & inserts a record into the watched index & then delete the job.
+        waitAndDeleteWatcherJob(schedJobParameter.getIndexToWatch(), jobId);
+        long actualCount = waitAndCountRecords(index, 180000);
```

Comment:
I think that I would feel good about dropping this down to like 130 seconds. Do you think there are any cases which the extra time would help to catch?